manifests/charts/istio-cni/values.yaml

```diff
@@ -44,6 +44,20 @@ cni:
   # Allow the istio-cni container to run in privileged mode, needed for some platforms (e.g. OpenShift)
   privileged: false
 
+  providers:
```

Comment:
I'm ok with this - but I was thinking of a simpler 

# Upstream CNI provider if known
# TODO: autodetect if possible with helm
provider: "default"

And if provider == multus, just set that thing on. Is there any reason to NOT set it when multus is used,
or to use a different namespace ? If not - no need to have another install option.